scipy/spatial/distance.py

```diff
@@ -1050,7 +1049,7 @@ def pdist(X, metric='euclidean', p=2, w=None, V=None, VI=None):
        Computes the Jaccard distance between the points. Given two
        vectors, ``u`` and ``v``, the Jaccard distance is the
        proportion of those elements ``u[i]`` and ``v[i]`` that
-       disagree where at least one of them is non-zero.
+       disagree.
```

Comment:
Oh, should I change it to "where at least one of them is not NaN"?  Or revert it back.  I'm not sure I fully get it, but I can change it back...